docs/docs-ref-conceptual/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -30,9 +31,283 @@ For more information about versions, build numbers, and Office Online Server, se
+|[chartPlotArea](/javascript/api/excel/excel.chartplotarea)|_Property_ > height|Represents the height value of plotArea.|1.8|
+|[chartPlotArea](/javascript/api/excel/excel.chartplotarea)|_Property_ > insideHeight|Represents the insideHeight value of plotArea.|1.8|
+|[chartPlotArea](/javascript/api/excel/excel.chartplotarea)|_Property_ > insideLeft|Represents the insideLeft value of plotArea.|1.8|
+|[chartPlotArea](/javascript/api/excel/excel.chartplotarea)|_Property_ > insideTop|Represents the insideTop value of plotArea.|1.8|
+|[chartPlotArea](/javascript/api/excel/excel.chartplotarea)|_Property_ > insideWidth|Represents the insideWidth value of plotArea.|1.8|
+|[chartPlotArea](/javascript/api/excel/excel.chartplotarea)|_Property_ > left|Represents the left value of plotArea.|1.8|
+|[chartPlotArea](/javascript/api/excel/excel.chartplotarea)|_Property_ > top|Represents the top value of plotArea.|1.8|
+|[chartPlotArea](/javascript/api/excel/excel.chartplotarea)|_Property_ > width|Represents the width value of plotArea.|1.8|
+|[chartPlotArea](/javascript/api/excel/excel.chartplotarea)|_Relationship_ > format|Represents the formatting of a chart plotArea. Read-only.|1.8|
+|[chartPlotArea](/javascript/api/excel/excel.chartplotarea)|_Relationship_ > position|Represents the position of plotArea.|1.8|
+|[chartPlotAreaFormat](/javascript/api/excel/excel.chartplotareaformat)|_Relationship_ > border|Represents the border attributes of a chart plotArea. Read-only.|1.8|
+|[chartPlotAreaFormat](/javascript/api/excel/excel.chartplotareaformat)|_Relationship_ > fill|Represents the fill format of an object, which includes background formating information. Read-only.|1.8|
+|[chartSeries](/javascript/api/excel/excel.chartseries)|_Property_ > explosion|Returns or sets the explosion value for a pie-chart or doughnut-chart slice. Returns 0 (zero) if there's no explosion (the tip of the slice is in the center of the pie). ReadWrite.|1.8|
+|[chartSeries](/javascript/api/excel/excel.chartseries)|_Property_ > firstSliceAngle|Returns or sets the angle of the first pie-chart or doughnut-chart slice, in degrees (clockwise from vertical). Applies only to pie, 3-D pie, and doughnut charts. Can be a value from 0 through 360. ReadWrite|1.8|
+|[chartSeries](/javascript/api/excel/excel.chartseries)|_Property_ > invertIfNegative|True if Microsoft Excel inverts the pattern in the item when it corresponds to a negative number. ReadWrite.|1.8|
+|[chartSeries](/javascript/api/excel/excel.chartseries)|_Property_ > overlap|Specifies how bars and columns are positioned. Can be a value between 100 and 100. Applies only to 2-D bar and 2-D column charts. ReadWrite.|1.8|
+|[chartSeries](/javascript/api/excel/excel.chartseries)|_Property_ > secondPlotSize|Returns or sets the size of the secondary section of either a pie of pie chart or a bar of pie chart, as a percentage of the size of the primary pie. Can be a value from 5 to 200. ReadWrite.|1.8|
```

Comment:
"a pie of pie chart"- a slice or portion ?
Also, unclear what is primary pie

Reply:
Sent a question to the Chart team. Not sure what's that supposed to say.